tests/internal/e2e/test/specs_config.ts

```diff
@@ -1,62 +1,187 @@
-export function getSpecGroups(vcast24: boolean) {
+export function getSpecGroups(useVcast24: boolean) {
   const specGroups = {
-    basic_user_interactions: [
-      "./**/**/vcast.build_env.test.ts",
-      "./**/**/vcast.create_script_1.test.ts",
-      "./**/**/vcast.create_script_2_and_run.test.ts",
-      "./**/**/vcast.create_second_test_1.test.ts",
-      "./**/**/vcast.create_second_test_2_and_run.test.ts",
-      "./**/**/vcast.third_test.test.ts",
-      "./**/**/vcast.rest.test.ts",
-      "./**/**/vcast.rest_2.test.ts",
-      "./**/**/vcast.rest_3.test.ts",
-    ],
-    bugs: [
-      "./**/**/vcast_testgen_bugs.test.ts",
-      "./**/**/vcast_testgen_bugs_2.test.ts",
-    ],
-    flask_icon: ["./**/**/vcast_testgen_flask_icon.test.ts"],
-    func_basis: [
-      "./**/**/vcast_testgen_func_basis.test.ts",
-      "./**/**/vcast_testdel_func_basis.test.ts",
-    ],
-    unit_basis: [
-      "./**/**/vcast_testgen_unit_basis.test.ts",
-      "./**/**/vcast_testdel_unit_basis.test.ts",
-    ],
-    env_basis: [
-      "./**/**/vcast_testgen_env_basis.test.ts",
-      "./**/**/vcast_testdel_env_basis.test.ts",
-    ],
+    basic_user_interactions: {
+      specs: [
+        "./**/**/vcast.build_env.test.ts",
+        "./**/**/vcast.create_script_1.test.ts",
+        "./**/**/vcast.create_script_2_and_run.test.ts",
+        "./**/**/vcast.create_second_test_1.test.ts",
+        "./**/**/vcast.create_second_test_2_and_run.test.ts",
+        "./**/**/vcast.third_test.test.ts",
+        "./**/**/vcast.rest.test.ts",
+        "./**/**/vcast.rest_2.test.ts",
+        "./**/**/vcast.rest_3.test.ts",
+      ],
+      env: {},
+      params: {},
+    },
+    build_env_failure: {
+      specs: [
+        "./**/**/vcast_build_env_failure.test.ts",
+        "./**/**/vcast_build_env_after_failure.test.ts",
+      ],
+      env: {
+        VECTORCAST_DIR_TEST_DUPLICATE: process.env.VECTORCAST_DIR,
+        VECTORCAST_DIR: "",
+        VECTORCAST_ATG_DIR: "",
```

Comment:
Let's get rid of `VECTORCAST_ATG_DIR` if it's never used (have you searched the repo?)

Reply:
Got rid of VECTORCAST_ATG_DIR in 3c5f99218af5dc6b962549004871a5fe6788d65a.